src/plugins/blocksmith/utils/index.js

```diff
@@ -33,6 +33,7 @@ import isFirstListBlockFn from './isFirstListBlock';
 import isLastListBlockFn from './isLastListBlock';
 import loadFacebookSDKFn from './loadFacebookSDK';
 import loadTwitterSDKFn from './loadTwitterSDK';
+import moveSelectionToEndOfBlockFn from './moveSelectionToEndOfBlock';
```

Comment:
@rsumilang there is a util for this already, `selectBlock`, use it with `selectionTypes.END` to do the same as `moveSelectionToEndOfBlock`